smart_open/tests/test_hdfs.py

```diff
@@ -17,6 +17,7 @@
 
 import smart_open.hdfs
 
+from sys import platform
```

Comment:
Avoid importing stuff directly into the module namespace (namespaces are a good thing, let's use them).

```suggestion
import sys
```